.prettierignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
I assume this is to prevent your editor from autoformatting? It would be nice to introduce prettier to this project in a future PR, but for now can you remove this file?

Reply:
Exactly. Okay :)